styles.css

```diff
@@ -0,0 +1,2153 @@
+    top: 14.5%;
+    background-image: url("images/header-shapes\ mobile.png");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    border-bottom-left-radius: 50px;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 114px 24px;
+    gap: 2px;
+    position: absolute;
+    width: 375px;
+    height: 604px;
+    left: calc(50% - 375px / 2);
+    top: -10%;
+  }
+
+  .primary-text1 {
+    width: 327px;
+    height: 150px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    margin-bottom: 0%;
+  }
+
+  .primary-text2 {
+    width: 327px;
+    height: 146px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  #br {
+    display: block;
+  }
+
+  .socialmedia {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 16px;
+    width: 200px;
+    height: 24px;
+  }
+
+  .supporting-text {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    width: 200px;
+    height: 24px;
+  }
+
+  .connect {
+    width: 200px;
+    height: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  #threesixty {
+    display: none;
+  }
+
+  .socialmedia-icons {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    width: 100%;
+    height: 40px;
+  }
+
+  .twitter {
+    position: absolute;
+    left: 6.4%;
+    right: 88.27%;
+    top: 63.15%;
+    bottom: 18.88%;
+  }
+
+  .linkedin {
+    position: absolute;
+    width: 20px;
+    height: 20px;
+    left: 17.47%;
+    top: 63.15%;
+  }
+
+  .m {
+    position: absolute;
+    left: 27.47%;
+    right: 67.2%;
+    top: 63.15%;
+    bottom: 19.21%;
+  }
+
+  .github {
+    position: absolute;
+    left: 38.13%;
+    right: 57.07%;
+    top: 63.15%;
+    bottom: 18.87%;
+  }
+
+  .vector {
+    position: absolute;
+    left: 48%;
+    right: 48.22%;
+    top: 63.15%;
+    bottom: 18.87%;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: auto;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    width: 100%;
+    height: 2788px;
+    position: absolute;
+    top: 44rem;
+  }
+
+  .card {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 15px 16px 55px;
+    gap: 12px;
+    width: 327px;
+
+    /* height: 552px; */
+    height: auto;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
```

Comment:
- [ ] the card item should take almost the whole width with a little amount of space on the left and right space as shown in the Figma design, I think if you work on removing the `position-absolute` mentioned before, then fixing this will be easy ✔️ 

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221351332-aa6f91af-1c6d-4a57-84f7-210090c6ea95.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/221351300-5b3b1107-8b19-4d8a-88f4-e6ca4d5f92ab.png)